src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -81,117 +112,278 @@ public void focus() {
+                At least one of the optional fields must be provided.
+                Existing values will be updated to the new input values.
+                When editing tags, the existing tags of the person will be removed (i.e., adding tags is not cumulative).
+                You can remove all the person’s tags by typing `t/` without specifying any tags after it.\n
+                Examples:
+                `edit 1 p/91234567 e/johndoe@example.com`: Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com`, respectively.
+                `edit 2 n/Betsy Crower t/`: Edits the name of the 2nd person to `Betsy Crower` and clears all existing tags.
+                """);
+
+
+        contentMap.put("Finding Contacts", """
+                Finding Contacts\n
+                Finds persons whose attributes contain any of the given keywords.\n
+                Format: `find KEYWORD [MORE_KEYWORDS]`\n
+                The search is case-insensitive. For example, `hans` will match `Hans`.
+                Keywords are matched against Name, Phone, Address, Email, Birthday, and Tag.
+                The order of the keywords does not matter. For example, `Hans Bo` will match `Bo Hans`.
+                Persons matching at least one keyword will be returned (i.e., OR search). For example, `Hans Bo` will return `Hans Gruber`, `Bo Yang`.
+                Partial matches of a keyword will also be returned. For example, `9123` will return the person with the phone number `91234567`.\n
+                Examples:
+                `find John` returns `john` and `John Doe`.
+                `find alex david` returns `Alex Yeoh`, `David Li`.
+                """);
+
+
+        contentMap.put("Deleting a Contact", """
+                Deleting a Contact\n
+                Deletes the specified person from the FART book.\n
+                Format: `delete INDEX`\n
+                Deletes the person at the specified INDEX.
+                The **INDEX** refers to the index number shown in the displayed person list.
+                The index must be a positive integer (1, 2, 3, …​).\n
+                Examples:
+                `list` followed by `delete 2` deletes the 2nd person in the FART book.
+                `find Betsy` followed by `delete 1` deletes the 1st person in the results of the find command.\n
+                Note: As the delete command cannot be undone, the FART Book will first prompt you to confirm that the contact you want to delete is correct. If you select Cancel, the deletion will be aborted.
+                """);
+
+        contentMap.put("Clearing All Entries", """
+                Clearing All Entries\n
+                Clears all entries from the FART book.\n
+                Format: `clear`\n
+                This command will remove all contacts from the FART book.
+                """);
+
+        contentMap.put("Marking a Person as Paid/Unpaid", """
```

Comment:
I think it's better to be consistent and choose to leave either 1 or 2 lines in between each contentMap.put